src/util/json.h

```diff
@@ -283,6 +283,12 @@ class json_objectt:public jsont
   {
   }
 
+  explicit json_objectt(
+    std::initializer_list<typename objectt::value_type> initializer_list)
+    : json_objectt{objectt{initializer_list}}
```

Comment:
Thank you for this further improvement! I suspect it's kind of a painful ask, but would you mind adding a commit that now removes the outermost parentheses from various bits touched in 64df79edce7dec4f?